templates/layouts/petition-and-scroll-to-donate.liquid

```diff
@@ -96,6 +96,11 @@
       petitionSidebar.fadeOut();
       petitionOverlayButton.fadeOut();
       displayAndScrollToYesNoQuestion();
+      fundraiserSignIn(data.petitionForm);
+    }
+
+    function fundraiserSignIn(data) {
+      storeDispatch({ type: 'login_member', payload: {formValues: data} });
```

Comment:
Preferably `window.champaign.store.dispatch` so it's easier to understand where it comes from.